google-cloud-translate/lib/google-cloud-translate.rb

```diff
@@ -62,8 +70,10 @@ module Cloud
     #   translation = translate.translate "Hello world!", to: "la"
     #   translation.text #=> "Salve mundi!"
     #
-    def translate key = nil, retries: nil, timeout: nil
-      Google::Cloud.translate key, retries: (retries || @retries),
+    def translate key = nil, scope: nil, retries: nil, timeout: nil
```

Comment:
Even though Translation now supports service accounts, both this method and the class method below have a different signature from other services. I think this is the correct choice since Translation continues to prioritize API key usage, and it is backward-compatible, but I just want to point this out.